### PR TITLE
Add reactions benchmarks

### PR DESCRIPTION
**Pull request by TarekkMA**

### CODE REVIEW

File: integration-tests/src/mock.rs [hunk 1]
```diff
@@ -129,6 +129,7 @@ impl pallet_profiles::Config for TestRuntime {
 
 impl pallet_reactions::Config for TestRuntime {
     type Event = Event;
+    type WeightInfo = pallet_reactions::weights::SubstrateWeight<TestRuntime>;
```
Comment (F3Joule) [inline integration-tests/src/mock.rs:132]:
```suggestion
    type WeightInfo = ();
```

File: pallets/reactions/Cargo.toml [hunk 2]
```diff
@@ -12,6 +12,7 @@ categories = ['cryptography::cryptocurrencies']
 
 [features]
 default = ['std']
+runtime-benchmarks = ["frame-benchmarking/runtime-benchmarks"]
```
Comment (F3Joule) [inline pallets/reactions/Cargo.toml:15]:
```suggestion
runtime-benchmarks = ['frame-benchmarking/runtime-benchmarks']
```

File: pallets/reactions/Cargo.toml [hunk 3]
```diff
@@ -26,6 +27,7 @@ std = [
     'pallet-spaces/std',
     'subsocial-support/std',
 ]
+try-runtime = ["frame-support/try-runtime"]
```
Comment (F3Joule) [inline pallets/reactions/Cargo.toml:30]:
```suggestion
try-runtime = ['frame-support/try-runtime']
```

File: pallets/reactions/Cargo.toml [hunk 4]
```diff
@@ -39,6 +41,7 @@ pallet-spaces = { default-features = false, path = '../spaces' }
 subsocial-support = { default-features = false, path = '../support' }
 
 # Substrate dependencies
+frame-benchmarking = { optional = true, git = "https://github.com/paritytech/substrate", branch = "polkadot-v0.9.24", default-features = false }
```
Comment (F3Joule) [inline pallets/reactions/Cargo.toml:44]:
```suggestion
frame-benchmarking = { git = 'https://github.com/paritytech/substrate', branch = 'polkadot-v0.9.24', default-features = false, optional = true }
```

**Review by F3Joule — CHANGES_REQUESTED** [review]

**Review by F3Joule — APPROVED** [review]

File: pallets/reactions/src/benchmarking.rs [hunk 5]
```diff
@@ -0,0 +1,104 @@
...
+        let post = create_dummy_post::<T>(origin.clone())?;
+        let reaction_kind = ReactionKind::Upvote;
+        let reaction_id = NextReactionId::<T>::get();
+
+    }: _(origin, post.id, reaction_kind)
+    verify {
+        ensure!(ReactionIdsByPostId::<T>::get(post.id) == vec![reaction_id], "Reaction is not found");
+        ensure!(
+            ReactionById::<T>::get(reaction_id)
+                .expect("Reaction not found")
```
Comment (olehmell) [inline pallets/reactions/src/benchmarking.rs:71]: I believe that we should improve the description for test.

Maybe 
First: "Reaction is not found by postId."
Seconde: "Reaction is not found."

**Review by olehmell — APPROVED** [review]

**Pull request merged**